applications/asset_tracker_v2/doc/sensor_module.rst

```diff
@@ -0,0 +1,117 @@
+When data sampling has been carried out, the :c:enum:`SENSOR_EVT_ENVIRONMENTAL_DATA_READY` event is sent from the module with the sampled environmental sensor values.
+
+.. note::
+   The nRF9160 DK does not have any external sensors.
+   If the sensor module is queried for sensor data when building for the DK, the event :c:enum:`SENSOR_EVT_ENVIRONMENTAL_NOT_SUPPORTED` is sent out by the module
+   upon data sampling.
+
+Motion detection
+================
+
+Motion is detected when acceleration in either X, Y or Z plane exceeds the configured threshold value.
+The threshold is set in one of two ways:
+
+* When receiving the :c:enum:`DATA_EVT_CONFIG_INIT` event after boot.
+  This event contains the default threshold value set by :ref:`CONFIG_DATA_ACCELEROMETER_THRESHOLD` or retrieved from flash.
+* When receiving the :c:enum:`DATA_EVT_CONFIG_READY` event.
+  This occurs when a new threshold value has been updated from cloud.
+
+Both events contains an accelerometer threshold value ``accelerometer_threshold`` in m/s2, present in the event structure.
+
+Motion detection in the module is enabled and disabled in turn to control the number of :c:enum:`SENSOR_EVT_MOVEMENT_DATA_READY` events that are sent out by the sensor module.
+This functionality acts as flow control.
+It prevents the sensor module from sending events to the rest of the application if the device is accelerating over the threshold for extended periods of time.
+
+The applicaton module controls this behavior through the :c:enum:`APP_EVT_ACTIVITY_DETECTION_ENABLE` and :c:enum:`APP_EVT_ACTIVITY_DETECTION_DISABLE` events.
+The sensor module will only send out a :c:enum:`SENSOR_EVT_MOVEMENT_DATA_READY` event if it detects movement and activity detection is enabled.
+
+.. note::
+   The DK does not have an external accelerometer.
+   However, you can use **Button 2** on the DK to trigger movement for testing purposes.
+
+.. note::
+   The accelerometer available in the Thingy:91 needs detailed per-use-case tuning in order to determine reliably which accelerometer readings are considered "motion"and which not.
+   This is beyond the scope of the generalized asset tracker framework that this application provides.
+   Therefore it's readings are not transmitted to the cloud, and only used to detect a binary active and inactive state.
+
+Configuration options
+*********************
+
+.. _CONFIG_SENSOR_THREAD_STACK_SIZE:
+
+CONFIG_SENSOR_THREAD_STACK_SIZE - Sensor module thread stack size
+   This option configures the sensor module's internal thread stack size.
+
+Module states
+*************
+
+The sensor module has an internal state machine with the following states:
+
+* ``STATE_INIT`` - The initial state of the module in which it awaits its initial configuation from the data module.
+* ``STATE_RUNNING`` - The module is initialized and can be queried for sensor data It will also emit :c:enum:`SENSOR_EVT_MOVEMENT_DATA_READY ` on movement.
```

Comment:
```suggestion
* ``STATE_RUNNING`` - The module is initialized and can be queried for sensor data. It will also send :c:enum:`SENSOR_EVT_MOVEMENT_DATA_READY` on movement.
```
The misplaced backtick (just one extra space before the closing one), caused the doc build failure.